
From: Stephen Tweedie <sct@redhat.com>

Use IS_RDONLY() instead of checking MS_RDONLY manually, for readability.

Signed-off-by: Stephen Tweedie <sct@redhat.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/ext3/ioctl.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff -puN fs/ext3/ioctl.c~ext3-online-resize-use-is_rdonly fs/ext3/ioctl.c
--- 25/fs/ext3/ioctl.c~ext3-online-resize-use-is_rdonly	Thu Sep 30 17:32:43 2004
+++ 25-akpm/fs/ext3/ioctl.c	Thu Sep 30 17:32:43 2004
@@ -183,7 +183,7 @@ flags_err:
 		if (!capable(CAP_SYS_RESOURCE))
 			return -EPERM;
 
-		if (sb->s_flags & MS_RDONLY)
+		if (IS_RDONLY(inode))
 			return -EROFS;
 
 		if (get_user(n_blocks_count, (__u32 *)arg))
@@ -204,7 +204,7 @@ flags_err:
 		if (!capable(CAP_SYS_RESOURCE))
 			return -EPERM;
 
-		if (inode->i_sb->s_flags & MS_RDONLY)
+		if (IS_RDONLY(inode))
 			return -EROFS;
 
 		if (copy_from_user(&input, (struct ext3_new_group_input *)arg,
_
